test/fixtures/dialects/snowflake/snowflake_inline_comment.sql

```diff
@@ -0,0 +1,6 @@
+//Snowflake Inline Comment
+
+SELECT 1;
+
+SELECT 1//Snowflake Inline
+;
```

Comment:
Since you're adding a new file here, should we not also add the other variants (`--`, `#`)

Reply:
Yeah - can do